tools/firtool/firtool.cpp

```diff
@@ -360,6 +360,25 @@ static cl::opt<bool> stripDebugInfo(
     cl::desc("Disable source locator information in output Verilog"),
     cl::init(false), cl::cat(mainCategory));
 
+// Build mode options.
+enum BuildMode { BuildModeDebug, BuildModeRelease };
+static cl::opt<BuildMode> buildMode(
+    "O", cl::desc("Controls how much optimization should be performed."),
```

Comment:
```suggestion
    "O", cl::desc("Controls how much optimization should be performed"),
```
I see that IMDCE does this also, but generally no trailing period on option descriptions.  Don't feel strongly about the policy as much as being consistent 👍 .

Reply:
thanks, removed!